hack/install-deps.sh

```diff
@@ -76,15 +76,12 @@ GOPATH=${TMPGOPATH}
 # and switch to specified  version.
 # Varset:
 # 1) Pkg name;
-# 2) Version.
-# 3) Repo name (optional);
+# 2) Version;
+# 3) Repo name;
 checkout_repo() {
   local -r pkg=$1
   local -r version=$2
-  local repo=${3:-""}
-  if [ -z "${repo}" ]; then
-    repo=${pkg}
-  fi
+  local -r repo=$3
```

Comment:
Since the callers are pretty consistent I debated doing something like:
```sh
local -r what=$1
eval `echo local -r pkg=\${${what}_PKG}`
eval `echo local -r version=\${${what}_VERSION}`
eval `echo local -r repo=\${${what}_REPO}`
```
But I figured that might be a bit too tricksy and left it as is.